test/unit/main/post/views/filters/post-filters.service.spec.js

```diff
@@ -89,6 +89,41 @@ describe('Post Filters Service', function () {
                 order_unlocked_on_top: 'true'
             });
         });
+        it ('if filter is represented of an array, only selected filter-value should be removed', function () {
```

Comment:
@Angamanga might not be related, but we should also make sure that when all filters are removed from the array, we should set the default instead. LMK what you think. The behavior seems to be working fine so it would be missing just that test.

Reply:
other than that, it looks nice !

Reply:
new default test Looks nice. GJ!